src/cloud/docker/docker-containers.md

```diff
@@ -49,7 +49,7 @@ For example, the following command starts the Docker configuration generator in
 ./vendor/bin/ece-docker build:compose --mode="developer" --php 7.2
 ```
 
-See [See Docker services containters] for details.
+See [Docker service containers] for details.
```

Comment:
This looks like it's missing something... Shouldn't it be followed by another set of brackets with the name of the link definition like the rest of the links in this topic?